index.html

```diff
@@ -0,0 +1,355 @@
+              </li>
+              <li class="my__icons__li">
+                <a href="#" class="my__icons__link"><img src="images/hand.svg" alt="hand icon" /></a>
+              </li>
+              <li class="my__icons__li">
+                <a href="https://twitter.com/oscardlacruz15" class="my__icons__link"><img src="images/twitter.png" alt="twitter icon" /></a>
+              </li>
+              <li class="my__icons__li">
+                <a href="#" class="my__icons__link"><img src="images/M.png" alt="M icon" /></a>
+              </li>
+            </ul>
+            <button type="button" class="btn clr-p-500">
+              Get my resume
+            </button>
+          </div>
+          <ul class="my__views">
+            <li class="my_languages">
+              <div class="my_languages_favourite">
+                <h3 class="my_languages_favourite__title clr-n-900">
+                  Languages
+                </h3>
+                <img class="my_languages_favourite__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+              <ul class="my_languages__bottom">
+                <li class="list">
+                  <img src="images/js.png" alt="javascript icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    JavaScript
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/html.png" alt="html icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    HTML
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/css.png" alt="css icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    CSS
+                  </p>
+                </li>
+              </ul>
+            </li>
+            <li class="my__technologies">
+              <div class="my__technologies__top">
+                <h3 class="my__technologies__top__title clr-n-900">
+                  Frameworks
+                </h3>
+                <img class="my__technologies__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+            <li class="my__skills">
+              <div class="my__skills__top">
+                <h3 class="my__skills__top__title clr-n-900">
+                  Skills
+                </h3>
+                <img class="my__skills__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+          </ul>
+        </section>
+        <section class="form-section" id="reach_me">
+            <div class="div-wrap">
+             <h2 class="form_t">Contact me</h2>
+             <p class="form-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```

Comment:
Nice work on making your app look like the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=56ISm4d7DE3nsZgj-0)😉 Let's even make it better by ensuring the `white` color is consistent between the contact section and the skill section as given in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=56ISm4d7DE3nsZgj-0)😉 Kindly see the image below😎😎
<img width="975" alt="between sections app" src="https://user-images.githubusercontent.com/64643995/221371669-392a0ae4-81df-451f-9eae-090ad09de001.png">

don't forget we are [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=56ISm4d7DE3nsZgj-0) in this milestone😉

Reply:
ok, i have fixed it!